front/app/services/pages.ts

```diff
@@ -5,27 +5,25 @@ import streams, { IStreamParams } from 'utils/streams';
 const apiEndpoint = `${API_PATH}/pages`;
 
 export type TFixedPage = 'information' | 'faq' | 'accessibility-statement';
+export const FIXED_PAGES = ['information', 'faq', 'accessibility-statement'];
+export const FIXED_PAGES_ALLOWED_TO_EDIT = ['information', 'faq'];
 
 export type TPolicyPage =
   | 'terms-and-conditions'
   | 'privacy-policy'
   | 'cookie-policy';
-
-export type TFooterPage = TFixedPage | TPolicyPage;
-
-export const FIXED_PAGES: TFixedPage[] = [
-  'information',
-  'faq',
-  'accessibility-statement',
-];
-
-export const POLICY_PAGES: TPolicyPage[] = [
+export const POLICY_PAGES = [
```

Comment:
These constants can't have types defined, because then we have type errors in places where they are used.

What we can do instead if define the type of their elements in the place they're used as you can see in the **front/app/containers/Admin/settings/pages/index.tsx** file, line 43 for example.